debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerStreamingChangeEventSource.java

```diff
@@ -237,4 +273,21 @@ private void updateStartScn() {
     public void commitOffset(Map<String, ?> offset) {
         // nothing to do
     }
+
+    // todo: brough tin for review, but has already been ported to error-handler retriable stuff
```

Comment:
/all, ignore this block, I'm going to be removing this section, I added this for my tracking some specific connector retry handling logic that was done one way on the older contributor's fork to how we do this upstream.